src/components/NavigationItems.js

```diff
@@ -80,7 +80,14 @@ const NavigationItems = ({
 };
 
 const NavIcon = ({ page }) => {
-  if (iconLibrary[page.displayName]) {
+  if (iconLibrary[page.displayName === 'Attend events']) {
+    return (
+      <FeatherIcon
+        className={styles.headerIcon}
+        name={iconLibrary[page.displayName]}
+      />
+    );
+  } else if (iconLibrary[page.displayName]) {
```

Comment:
```suggestion
  if (iconLibrary[page.displayName]) {
```

Since we are using the arrow icon, we can revert this back to the way it was.